clients/fides-js/src/components/fides.css

```diff
@@ -319,6 +312,10 @@ button.fides-banner-button.fides-banner-button-tertiary {
   line-height: 1.25em;
 }
 
+button.fides-banner-button.fides-acknowledge-button {
```

Comment:
This class existed before- looks like we were using `168px` before. Is there a strong reason to change this?

Reply:
as mentioned above, it was never actually being applied. The old was also a 'max-width' before which didn't make much sense. I'm using a 'min-width' to make the short "Ok" string not look to small, and matching Jack's designs in Figma which used 160px.

Reply:
Jack's latest design had 160 so when I made the change mentioned above, I also went with his size.